_posts/2021-07-01-release-1.13.2.md

```diff
@@ -0,0 +1,348 @@
+---
+layout: post
+title:  "Apache Flink 1.13.2 Released"
+date:   2021-07-14 00:00:00
+categories: news
+authors:
+- tangyun:
+  name: "Yun Tang"
+
+---
+
+The Apache Flink community released the second bugfix version of the Apache Flink 1.13 series.
+
+This release includes 82 fixes and minor improvements for Flink 1.13.2. The list below includes bugfixes and improvements. For a complete list of all changes see:
```

Comment:
```suggestion
This release includes 82 fixes and minor improvements for Flink 1.13.1. The list below includes bugfixes and improvements. For a complete list of all changes see:
```

Reply:
This release is for `1.13.2` not for `1.13.1`.